doc/lua_api.txt

```diff
@@ -4239,6 +4239,28 @@ Definition tables
         ^ Shall drop item and return the leftover itemstack
         ^ The dropper may be any ObjectRef or nil.
         ^ default: minetest.item_drop ]]
+        on_pickup = func(item_luaentity, up_picker, time_from_last_punch, tool_capabilities, dir),
+        --[[
+        ^ Called when a dropped item is punched and shall be picked up.
```

Comment:
'and shall be' is not good English, do you mean to explain this is called before the item is added to the inventory?